app/controllers/ncr/work_orders_controller.rb

```diff
@@ -4,7 +4,6 @@ class WorkOrdersController < ClientDataController
     MAX_UPLOADS_ON_NEW = 10
 
     def new
-      work_order.approving_official_email = suggested_approver_email
```

Comment:
adding this line back in unfortunately does not fix the regression noted in the PR description:

> we are no longer auto-suggesting the approver from the last proposal

I am not sure why, but wanted to debug separately since this PR was getting larger. Note that the `edit` action still auto populates the existing approving official email in the form.